### PR TITLE
pending transactions management

### PR DESCRIPTION
**Pull request by IViol**

### CODE REVIEW

File: src/components/AddressPicker/Current/current.scss [hunk 1]
```diff
@@ -1,5 +1,3 @@
-@import 'styles/mixins';
```
[the remark this answers is not in the document]
Comment (IViol) [reply src/components/AddressPicker/Current/current.scss]: this syntax cause error during storybook build

File: src/components/DoubleInput/doubleInput.scss [hunk 2]
```diff
@@ -1,5 +1,3 @@
-@import 'styles/mixins';
```
[the remark this answers is not in the document]
Comment (IViol) [reply src/components/DoubleInput/doubleInput.scss]: fast fix for storybook

File: src/components/TransactionItem/Main/Main.js [hunk 3]
```diff
@@ -35,6 +36,12 @@ type Props = {|
   +isAssetList: boolean,
 |}
 
+const ONE_DAY: number = 1000 * 60 * 60 * 24
```
[the remark this answers is not in the document]
Comment (IViol) [reply src/components/TransactionItem/Main/Main.js]: actually it's for mining delay. To notify user that his transaction is still not mined and show red icon.

File: src/components/DoubleInput/doubleInput.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (stcherenkov) [reply src/components/DoubleInput/doubleInput.scss]: I added it as a bug: https://jibrelnetwork.atlassian.net/browse/JWWEB-409

File: src/config.js [hunk 4]
```diff
@@ -149,6 +151,8 @@ const config: AppConfig = {
   resyncTransactionsTimeout: 5 * ONE_MINUTE,
 
   syncBalancesTimeout: 10 * ONE_SECOND,
+
+  miningDelay: ONE_DAY,
```
Comment (stcherenkov) [inline src/config.js]: It looks more like "timeout" (if it is not mined in specified time, it is considered an error), but right now I'm fine with "delay" too

**Pull request merged**